application/account-management/Api/AccountRegistrations/AccountRegistrationsEndpoints.cs

```diff
@@ -4,11 +4,11 @@
 
 namespace PlatformPlatform.AccountManagement.Api.AccountRegistrations;
 
-public static class AccountRegistrationsEndpoints
+public class AccountRegistrationsEndpoints : IEndpoints
```

Comment:
Would there be a way to add `ISender mediator` as dependency, so we don't have to inject it into all methods.

I understand that one of the cool things about Minimal API is that you do method injection, but in PlatformPlatform the MediatR is the only dependency on all methods, so it would simplify the method signatures if we could move it to a class dependency.

Reply:
I don't really follow what you mean here. Can you explain a bit more what you need we want to achieve?